lib/agenda/db-init.js

```diff
@@ -12,19 +12,22 @@ module.exports = function(collection, cb) {
   debug('init database collection using name [%s]', collection);
   this._collection = this._mdb.collection(collection || 'agendaJobs');
   debug('attempting index creation');
```

Comment:
Slightly misleading 😂

Reply:
I'll update this to be more appropriate.